purchase_minimum_amount/__init__.py

```diff
@@ -1,4 +1,3 @@
-# Copyright 2016 Eficent Business and IT Consulting Services S.L.
```

Comment:
Why removing copyrights?

Reply:
@jarroyomorales  no need copyrights in init file.

Reply:
True, sorry